### DISCUSSION TITLE
When I deploy skywalking oap and set the two oap servers as receiver and aggregator roles, which configuration item should I use to configure data transmission between the two roles?

### DISCUSSION THREAD

**Discussion by iliujiannan** [apache/skywalking#10668] [Q&A] [answered]
When I deploy skywalking oap and set the two oap servers as receiver and aggregator roles, which configuration item should I use to configure data transmission between the two roles?

**Comment by wu-sheng** [comment, marked as answer]
You just need to put all OAP in one cluster, no extra configuration is required. Just need to be sure, the cluster configuration should be correct.

**Reply by iliujiannan** [reply]
Can this deployment scheme only be used in cluster deployment mode? What if both OAP servers are deployed using standalone mode?

**Reply by wu-sheng** [reply]
No, that is totally wrong.

**Reply by iliujiannan** [reply]
I feel quite confused about the 'cluster configuration' you mentioned before, could you please provide some detail info about this?

**Reply by wu-sheng** [reply]
I don't know what you want. Those two roles are belonging to one cluster. Nothing special, nothing to provide.
